src/App.tsx

```diff
@@ -1,28 +1,56 @@
-/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import React, { useEffect, useMemo, useState } from 'react';
+import classNames from 'classnames';
+import { getTodos } from './api/todos';
+import { TodoList } from './components/Todo';
+import { Todo } from './types/Todo';
+import { Filter } from './types/Filter';
+import { getFilteredTodos } from './utils/TodosFilter';
+import { TodoFilter } from './components/TodoFilter/TodoFilter';
+import { Error } from './types/Error';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [sortedStatus, setSortedStatus] = useState<Filter>(Filter.ALL);
+  const [errorMessage, setErrorMessage] = useState<string>('');
+
+  useEffect(() => {
+    getTodos()
+      .then(setTodos)
+      .catch(error => {
+        setErrorMessage(Error.GET);
+        throw error;
```

Comment:
You don't to throw error, you set it to the state, show to user and that's enough